src/main/java/com/shapesecurity/salvation2/Values/MediaType.java

```diff
@@ -24,8 +24,9 @@ public static Optional<MediaType> parseMediaType(String value) {
 		if (matcher.find()) {
 			// plugin type matching is ASCII case-insensitive
 			// https://w3c.github.io/webappsec-csp/#plugin-types-post-request-check
-			String type = matcher.group("type").toLowerCase(Locale.ENGLISH);
-			String subtype = matcher.group("subtype").toLowerCase(Locale.ENGLISH);
+			// Converted to .group(int) for use with TeaVM
```

Comment:
I don't think you need these TeaVM comments all over the place.

Reply:
I'll remove, but I put them there in case someone asks "why not use named groups?"

Reply:
I think that's solved by putting that restriction in the README (and having tests that would catch it in CI), not by putting a comment at every use of a capturing group.